docs/ndk/guide/env/python.md

```diff
@@ -33,17 +37,16 @@ sudo python3 -m venv /opt/myApp/venv
 # activate the newly created venv
 source /opt/myApp/venv/bin/activate
 
-# update pip/setuptools in the venv
-pip3 install -U pip setuptools
```

Comment:
why do you think we need to remove updating pip (and setuptools)?

Reply:
Looking back this should have remained, adding it back